eks-worker-al2.json

```diff
@@ -22,8 +24,10 @@
     "kernel_version": "",
     "kms_key_id": "",
     "kubernetes_build_date": null,
+    "kube_proxy_version_suffix": "eksbuild.1",
     "kubernetes_version": null,
```

Comment:
Can we ensure that `eksbuild.1` always exist?
I'll worry about a user set  `kubernetes_version ` but forget to change `kube_proxy_version_suffix`?

Reply:
Take current state as example, it's now on `eksbuild.2`.
ref: https://docs.aws.amazon.com/eks/latest/userguide/managing-kube-proxy.html